src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/refreshanalyzer/RefreshSearchAnalyzerAction.kt

```diff
@@ -20,7 +20,7 @@ import org.elasticsearch.common.io.stream.Writeable
 
 class RefreshSearchAnalyzerAction : ActionType<RefreshSearchAnalyzerResponse>(NAME, reader) {
     companion object {
-        const val NAME = "indices:admin/refresh_search_analyzer"
+        const val NAME = "indices:admin/_refresh_search_analyzers"
```

Comment:
Is it common to prefix _ on the action names?

Reply:
You're right, it's not. Will change back to `indices:admin/refresh_search_analyzers`